doc/book/zend.barcode.objects.md

```diff
@@ -0,0 +1,403 @@
+|barThickWidth     |Integer                          |3                    |Width of the thick bar                                                                                     |
+|barThinWidth      |Integer                          |1                    |Width of the thin bar                                                                                      |
+|factor            |Integer, Float, String or Boolean|1                    |Factor by which to multiply bar widths and font sizes (barHeight, barThinWidth, barThickWidth and fontSize)|
+|foreColor         |Integer                          |0x000000 (black)     |Color of the bar and the text. Could be provided as an integer or as a HTML value (e.g. "#333333")         |
+|backgroundColor   |Integer or String                |0xFFFFFF (white)     |Color of the background. Could be provided as an integer or as a HTML value (e.g. "#333333")               |
+|orientation       |Integer, Float, String or Boolean|0                    |Orientation of the barcode                                                                                 |
+|font              |String or Integer                |NULL                 |Font path to a TTF font or a number between 1 and 5 if using image generation with GD (internal fonts)     |
+|fontSize          |Float                            |10                   |Size of the font (not applicable with numeric fonts)                                                       |
+|withBorder        |Boolean                          |FALSE                |Draw a border around the barcode and the quiet zones                                                       |
+|withQuietZones    |Boolean                          |TRUE                 |Leave a quiet zone before and after the barcode                                                            |
+|drawText          |Boolean                          |TRUE                 |Set if the text is displayed below the barcode                                                             |
+|stretchText       |Boolean                          |FALSE                |Specify if the text is stretched all along the barcode                                                     |
+|withChecksum      |Boolean                          |FALSE                |Indicate whether or not the checksum is automatically added to the barcode                                 |
+|withChecksumInText|Boolean                          |FALSE                |Indicate whether or not the checksum is displayed in the textual representation                            |
+|text              |String                           |NULL                 |The text to represent as a barcode                                                                         |
+
+ ..
+\_zend.barcode.barcode.common.options.barcodefont:
```

Comment:
This is another of the reference markers. If they're coming in like this, we can likely strip them with a regex like the following:

```
/[^|\n]\.\.\s*\n\\_[a-z0-9._-]+:\s*\n/s
```

Use that as the match argument of a `preg_replace`, and make the replacement simply `"\n"`, and it should be good to go.